flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/FlinkConfMountDecorator.java

```diff
@@ -182,8 +182,7 @@ String getFlinkConfData(Map<String, String> propertiesMap) throws IOException {
 		return localLogConfFiles;
 	}
 
-	@VisibleForTesting
-	String getFlinkConfConfigMapName(String clusterId) {
+	public static String getFlinkConfConfigMapName(String clusterId) {
```

Comment:
```suggestion
    @VisibleForTesting
	public static String getFlinkConfConfigMapName(String clusterId) {
```

Still it is a private method while we expose it for testing.